components/button/buttonStyle.css

```diff
@@ -0,0 +1,77 @@
+:root {
+    --primary-color: #319795;
+    --white: #fff;
+    --second-color: #89ebe9;
+}
+
+.button1 {
+    background-color: var(--primary-color);
+    color: var(--white);
+    font-size: 1rem;
+    padding: 1rem;
+    border: none;
+    border-radius: 0.75rem;
+    margin: 1rem;
+}
+.button1:hover {
+    background-color: var(--second-color);
+}
+
+.button2 {
+    background-color: transparent;
+    color: var(--primary-color);
+    font-size: 1rem;
+    padding: 1rem;
+    border-color: var(--primary-color);
+    border-radius: 0.75rem;
+    margin: 1rem;
+}
+.button2:hover {
+    background-color: #89ebe9;
+}
+
+.button3 {
+    background-color: transparent;
+    color: #319795;
+    font-size: 1rem;
+    padding: 1rem;
+    border-color: #319795;
+    border-radius: 0.75rem;
+    margin: 1rem;
+}
+.icon {
+    padding-right: 1rem;
+}
+.button3:hover {
+    background-color: var(--second-color);
+}
+
+.button4 {
+    background-color: transparent;
+    font-size: 1rem;
+    padding: 1rem;
+    border: none;
+    margin: 1rem;
+}
+.link{
+    color: var(--second-color);
+}
+.link:hover {
+    color: var(--primary-color);
+}
+
+.button5 {
+    background-color: transparent;
+    color: var(--primary-color);
+    font-size: 2rem;
+    padding: 0.25rem;
+    height: 3rem;
+    width: 3rem;
+    overflow: hidden;
+    border-radius: 50%;
+    margin: 1rem;
+}
+
+.button5:active {
+    float: right;
+}
```

Comment:
why have you used float here? i checked in csb, looks like a mistake to me.

Reply:
This one is resolved Thank you